### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 3 - Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#518] [Esercizi M-Z]
![es.4](https://github.com/user-attachments/assets/a5bd5ec9-49e9-4395-8fbd-94607016b038)

**Comment by AglaiaNorza** [comment]
(fatto in classe dal professor Pellarin)
![image](https://github.com/user-attachments/assets/87b11c14-f2eb-4e77-8494-7267a5b39327)

**Comment by marcoseverini** [comment]
<img width="1003" alt="immagine" src="https://github.com/user-attachments/assets/f85940b0-44c8-428f-bb18-baeb7f1020c8">
